src/ops/ops.go

```diff
@@ -744,6 +744,7 @@ func (o *ops) CreateManifests(kubeconfig string, content []byte) error {
 	defer os.Remove(file.Name())
 
 	// Write the content to the temporary file:
+	// #nosec
```

Comment:
I wasn't sure of the implications of changing actual product code, so I just disabled it for now